### PR TITLE
Sp2 Front End Create Campaign, Ad Group, Tile Navigation 2

### PR DESCRIPTION
**Pull request by tkiethanom**
r? @rlr

You can now Create Campaigns, Ad Groups and Tiles from the Navigation. Dropdowns to select parent Model now appear in the form.

Ad Group Channel removed, value is pulled from parent Campaign #237

Now using react-select instead of select2 #240

### CODE REVIEW

File: front_end/package.json [hunk 1]
```diff
@@ -74,6 +74,7 @@
     "react-mixin": "^3.0.1",
     "react-redux": "^4.0.0",
     "react-router": "1.0.0",
+    "react-select": "^0.9.1",
```
Comment (rlr) [inline front_end/package.json]: can we remove select2 now?

**Comment by rlr** [comment]
much better now. travis isnt happy:

```
5.57s$ cd front_end && npm install && npm test && cd ..

npm WARN package.json babel-core@5.8.22 No README data

npm WARN package.json react-addons-css-transition-group@0.14.0 No repository field.

-

> radium@0.13.8 postinstall /home/travis/build/oyiptong/splice/front_end/node_modules/radium

> node -e "require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})" || npm run lib

npm ERR! peerinvalid The package react does not satisfy its siblings' peerDependencies requirements!

npm ERR! peerinvalid Peer react-addons-css-transition-group@0.14.0 wants react@^0.14.0

npm ERR! peerinvalid Peer react-dropzone@3.0.0 wants react@^0.14.0

npm ERR! peerinvalid Peer react-redux@4.0.0 wants react@^0.14.0

npm ERR! peerinvalid Peer react-select@0.9.1 wants react@^0.14.1

npm ERR! System Linux 3.13.0-40-generic

npm ERR! command "/home/travis/.nvm/v0.10.36/bin/node" "/home/travis/.nvm/v0.10.36/bin/npm" "install"

npm ERR! cwd /home/travis/build/oyiptong/splice/front_end

npm ERR! node -v v0.10.36

npm ERR! npm -v 1.4.28

npm ERR! code EPEERINVALID

npm ERR! 

npm ERR! Additional logging details can be found in:

npm ERR!     /home/travis/build/oyiptong/splice/front_end/npm-debug.log

npm ERR! not ok code 0
```

**Comment by tkiethanom** [comment]
peerinvalid errors fixed

**Comment by rlr** [comment]
Now I see this when firing up the server:

```
ERROR in ./src/main.js
Module build failed: TypeError: Error while loading rule 'react/no-multi-comp': Can't add property react, object is not extensible
    at Function.componentRule (/Users/rlr/dev/splice/front_end/node_modules/eslint-plugin-react/lib/util/Components.js:111:17)
    at /Users/rlr/dev/splice/front_end/node_modules/eslint/lib/eslint.js:692:28
    at Array.forEach (native)
    at EventEmitter.module.exports.api.verify (/Users/rlr/dev/splice/front_end/node_modules/eslint/lib/eslint.js:671:16)
    at processText (/Users/rlr/dev/splice/front_end/node_modules/eslint/lib/cli-engine.js:230:27)
    at CLIEngine.executeOnText (/Users/rlr/dev/splice/front_end/node_modules/eslint/lib/cli-engine.js:686:26)
    at lint (/Users/rlr/dev/splice/front_end/node_modules/eslint-loader/index.js:26:20)
    at Object.module.exports (/Users/rlr/dev/splice/front_end/node_modules/eslint-loader/index.js:127:7)
 @ multi main
```

**Comment by tkiethanom** [comment]
eslint@1.10.0 just got released today, looks like its creating some issues. Another good reason to lock these packages.

**Comment by tkiethanom** [comment]
Merged the dev dependency changes

**Comment by rlr** [comment]
OK. Now `npm install fails` when doing a clean install:

```
npm ERR! Darwin 15.0.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v4.2.1
npm ERR! npm  v2.14.7
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package react@0.14.1 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-addons-css-transition-group@0.14.3 wants react@^0.14.3
npm ERR! peerinvalid Peer react-dropzone@3.0.0 wants react@^0.14.0
npm ERR! peerinvalid Peer react-redux@4.0.0 wants react@^0.14.0
npm ERR! peerinvalid Peer react-select@0.9.1 wants react@^0.14.1

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/rlr/dev/splice/front_end/npm-debug.log
```

**Comment by tkiethanom** [comment]
@rlr try it again. Thanks!

**Comment by rlr** [comment]
awesome! r+! thank you!

**Pull request merged**